### PR TITLE
add app-template (aleixcam)

### PR DESCRIPTION
**Pull request by aleixcam**

### CODE REVIEW

File: staff/aleix-caminal/app-template/news.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    <li><a href="#culture">Culture</a></li>
+                    <li><a href="#world">World</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <section id="politics">
+                <h2>Sports</h2><hr>
+                <article>
+                    <h3>Lorem ipsum</h3>
```
Comment (almuitur) [inline staff/aleix-caminal/app-template/news.html:23]: Why not using header for news?

Comment (aleixcam) [reply staff/aleix-caminal/app-template/news.html:23]: just a title works for this i guess

File: staff/aleix-caminal/app-template/about.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <body>
+        <header>
+            <h1>About</h1><hr>
+        </header>
+        <main>
+            <section id="us">
+                <h2>Aleix Caminal</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </section>
+            <section id="services">
```
Comment (manuelbarzi) [inline staff/aleix-caminal/app-template/about.html:16]: why ids? are they really necessary? think about it

File: staff/aleix-caminal/app-template/contact.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </head>
+    <body>
+        <header>
+            <h1>Contact</h1><hr>
+        </header>
+        <main>
+            <form action="contact.html" method="post">
+                <section id="reasons">
+                    <h2>Reasons for inquiry</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (manuelbarzi) [inline staff/aleix-caminal/app-template/contact.html:15]: shouldn't "reasons" be a select element with options to choose?

Comment (aleixcam) [reply staff/aleix-caminal/app-template/contact.html:15]: It's a dummy title

File: staff/aleix-caminal/app-template/about.html (hunk 2 above)
Comment (aleixcam) [reply staff/aleix-caminal/app-template/about.html:16]: They allow you to navigate between sections and differentiate them, I find them useful

**Pull request merged**